### PR TITLE
Allow path field in update

### PR DESCRIPTION
**Pull request by rambleraptor**
Allow path field in Update to fix errors on aepc

### CODE REVIEW

File: rules/aep0134/request_unknown_fields.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ var unknownFields = &lint.MessageRule{
 		allowedFields := stringset.New(
 			fieldNameFromResource(resource), // AEP-134
 			"allow_missing",                 // AEP-134
+			"path",                          // AEP-134
```
Comment (toumorokoshi) [inline rules/aep0134/request_unknown_fields.go:37]: can a linter rule be added to make sure path is required? Maybe re-using the get / delete one?

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
I'm ok if you want to follow up with a rule requiring path, but it'd be great to add that with this PR!

File: rules/aep0134/request_unknown_fields.go (hunk 1 above)
Comment (rambleraptor) [reply rules/aep0134/request_unknown_fields.go:37]: Added!

File: docs/rules/0134/request-path-required.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+rule:
+  aep: 134
+  name: [core, '0134', request-path-required]
+  summary: Delete RPCs must have a `path` field in the request.
```
Comment (toumorokoshi) [inline docs/rules/0134/request-path-required.md:5]:
```suggestion
  summary: Update RPCs must have a `path` field in the request.
```

I think a few more places need "Delete" replaced with "Update".

Comment (rambleraptor) [reply docs/rules/0134/request-path-required.md]: Potentially a way to do that? I can make a bug for it.

Comment (toumorokoshi) [reply docs/rules/0134/request-path-required.md]: I'd probably refactor the linter rule to look at specific method names (Read{singular}/Update{Singular}, etc), then require the field for those methods.

Maybe it'll be kind of hard to incorporate that into multiple different rule sets though.

**Pull request merged**